ui/src/validation.ts

```diff
@@ -0,0 +1,62 @@
+import validatejs from "validate.js"
+import { derived, writable, Readable, get } from "svelte/store"
+
+export enum ValidationStatus {
+  NotStarted = "NOT_STARTED",
+  FormatValidation = "FORMAT_VALIDATION",
+  Pending = "PENDING",
```

Comment:
Should we be consistent with `Remote` and call it `Loading`, or rename the state in `Remote`?

Reply:
`Loading` works